openwisp_ipam/static/openwisp-ipam/js/subnet.js

```diff
@@ -58,6 +59,41 @@ function initHostsInfiniteScroll($, current_subnet, address_add_url, address_cha
         });
         return div;
     }
+    async function validateIp(ip_address) {
```

Comment:
We can't use ES6 syntax (async/await) here because we don't have a transpiler in place and it would not work on all browsers

Reply:
Thank you, I have replaced that with callback now.